client/src/containers/AddTrip/index.js

```diff
@@ -10,11 +9,17 @@ class AddTrip extends Component {
             author: '',
             review: '',
             duration: '',
-            rating: '',
+            rating: '1',
             price: ''
         }
-    }
+    };
 
+    componentWillReceiveProps(newProps) {
+        const { newtrip } = newProps.trips;
```

Comment:
Usually, it is better to try to leave one empty line after a block of const/let declarations...
In this case, I do not want to create a commit to add one new line, but in the future it is better to follow this pattern.